src/App.vue

```diff
@@ -0,0 +1,26 @@
+<template lang='pug'>
+  .app
+    Form
+</template>
+
+<script>
+import Form from './components/Form.vue'
+
+export default {
+  name: 'App',
+  components: {
+    Form
+  }
+}
+</script>
+
+<style>
+#app {
+  font-family: Avenir, Helvetica, Arial, sans-serif;
+  -webkit-font-smoothing: antialiased;
```

Comment:
It would be cleaner to indent here.

Reply:
where? please comment on the line(s) in question